web/src/api/stationAPI.ts

```diff
@@ -15,6 +15,23 @@ export interface GeoJsonStation {
   properties: StationProperties
   geometry: StationGeometry
 }
+
+export interface DetailedGeoJsonStation {
+  type: string
+  properties: DetailedStationProperties
+  geometry: StationGeometry
+}
+
+export interface DetailedStationProperties extends StationProperties {
```

Comment:
I'm confused - how are weather values a property of the station? I'd like to see a comment explaining this.

Reply:
These are weather values measured at the station, I can add that if it makes it clearer